api/v1/batchtransformjob/batchtransformjob_types.go

```diff
@@ -56,7 +56,7 @@ type BatchTransformJobSpec struct {
 	Region *string `json:"region"`
 
 	// A custom SageMaker endpoint to use when communicating with SageMaker.
-	// +kubebuilder:validation:Pattern=^(https|http)://.*$
+	// +kubebuilder:validation:Pattern="^(https|http)://.*$"
```

Comment:
Were you able to verify that was actually a version mismatch issue ?

Reply:
Yes. This error was exactly as predicted.